README.rst

```diff
@@ -1,41 +1,230 @@
-# MVS - Multi-Vector Simulator of the E-Land toolbox
+==================================================
+MVS - Multi-Vector Simulator of the E-Land toolbox
+==================================================
 
 [![Documentation Status](https://readthedocs.org/projects/multi-vector-simulator/badge/?version=latest)](https://multi-vector-simulator.readthedocs.io/en/latest/?badge=latest)
 [![Build Status](https://travis-ci.com/rl-institut/mvs_eland.svg?branch=dev)](https://travis-ci.com/rl-institut/mvs_eland)
 [![Coverage Status](https://coveralls.io/repos/github/rl-institut/multi-vector-simulator/badge.svg)](https://coveralls.io/github/rl-institut/multi-vector-simulator)
 
 Rights: [Reiner Lemoine Institut (Berlin)](https://reiner-lemoine-institut.de/)
```

Comment:
Can you change this to proper `.rst` file links? Looks like this: `Name <link>`__ (two underscores).

Reply:
Done